librz/core/cmd/cmd_api.c

```diff
@@ -187,7 +185,25 @@ RZ_API void rz_cmd_alias_init(RzCmd *cmd) {
 	cmd->aliases.values = NULL;
 }
 
-RZ_API RzCmd *rz_cmd_new(bool has_cons) {
+static void macro_free(RzCmdMacro *macro) {
+	if (!macro) {
+		return;
+	}
+	free(macro->name);
+	free(macro->code);
+	for (size_t i = 0; i < macro->nargs; i++) {
+		free(macro->args[i]);
+	}
+	free(macro->args);
+	free(macro);
+}
+
+static void free_macro_kv(HtPPKv *kv) {
+	free(kv->key);
+	macro_free(kv->value);
+}
+
+RZ_API RzCmd *rz_cmd_new(RzCore *core, bool has_cons) {
```

Comment:
Looks similar to `rz_core_cmd_new()` - maybe noting the differences in the corresponding Doxygen descriptions could help